stackslib/src/clarity_cli.rs

```diff
@@ -703,6 +703,15 @@ impl HeadersDB for CLIHeadersDB {
         }
     }
 
+    fn get_block_time_for_block(&self, id_bhh: &StacksBlockId) -> Option<u64> {
+        let conn = self.conn();
+        if let Some(height) = get_cli_block_height(&conn, id_bhh) {
+            Some((height * 10 + 1713799973) as u64)
```

Comment:
What's with the magic timestamp here?

Reply:
Just wanted to use a real-ish value like the other method did. This is the timestamp of the 2.5 activation block. 😁